workshop/content/60-construct-hub/_index.md

```diff
@@ -0,0 +1,38 @@
+
+The Construct Hub is the central location where CDK users can find a comprehensive collection of constructs to help them build their applications. The Construct Hub makes it easier for developers to find the high-level building blocks they need to build their applications by listing publicly released construct libraries.
+
+## Discovering and Using Constructs
+Browse to https://constructs.dev and search for constructs based on keywords such as names of AWS services used (e.g., “eks”, “dynamodb”), the library’s author (e.g., “pahud”), or the target provisioning engine (e.g., “cdktf”, “cdk8s”). Examples of constructs you can find include datadog-cdk-constructs that instruments Python and Node.js Lambda functions with Datadog, cdk-gitlab-runner that creates a GitLab Runner and executes a pipeline job, cdk-k3s-cluster that deploys a K3s cluster, and many more. The Construct Hub website also includes links to Getting Started resources. Below is an example of search results for constructs that include the “bucket” keyword and support the Python programming language:
+
+Construct libraries listed on the Construct Hub include a detail page with instructions on how to install the package (click “Use Construct”) and an API reference that describes all the classes, interfaces, enums and data types in this library. The API reference and code samples are displayed in the selected programming language and are automatically generated from type information produced by the JSII compiler (JSII is a TypeScript-based programming language for creating multi-language libraries).
+
+## Listing Constructs on the Hub
+In order for your construct to be listed on the Construct Hub, you need to make sure it is authored with the following criteria:
+
+* Published to the [npmjs.com](https://npmjs.com/) registry
+* Uses a permissive license (Apache, BSD, MIT)
+* Annotated with one of the supported keywords (awscdk, cdk8s or cdktf)
+* Compiled with [JSII](https://aws.github.io/jsii/)
+
+As a construct library publisher, you can improve the presentation of your construct library on the Construct Hub by:
+
+* Adding links to the construct source code and documentation
+* Including a README file with usage instructions
+* Adding relevant keywords that will be displayed on the package page and can be used for search
+
+All constructs are owned by the publishers of the packages. Constructs are user-generated content that are governed by their own license terms which is displayed in the search results and can be accessed directly through the hyperlinked package page
```

Comment:
Please use the same language we used in constructs.dev/faq and constructs.dev/contribute for these issues about contribution, ownership and license